packages/components/src/click_away/index.jsx

```diff
@@ -0,0 +1,73 @@
+import { createContext, useCallback, useContext, useEffect, useRef } from 'react';
+
+const context = createContext(() => {
+  throw new Error('ClickAwayProvider not set up');
```

Comment:
Not clear, tbh, why do we need it here?

Reply:
Let's say some other component in `@goodhood/components` is using the useClickAway. If you don't specify anything in here, then it will just silently not work. Users of those libraries might not know that they are expected to use `ClickAwayProvider`